src/content/en/updates/posts/2016/04/intersectionobserver.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: updates/post
+title: "IntersectionObserver"
```

Comment:
I'd go with a catchier title.

"Watch Out for IntersectionObserver" or "IntersectionObserver's Coming into View" or something along those lines.